skfda/representation/_functional_data.py

```diff
@@ -788,6 +788,34 @@ def concatenate(self, *others, as_coordinates=False):
         """
         pass
 
+    @staticmethod
+    def concatenate_samples(objects):
+        """
+        Join samples from an iterable of similar FDataBasis objects.
+
+        Joins samples of FDataBasis objects if they have the same
+        dimensions and sampling points.
+        Args:
+            objects (list of :obj:`FDataBasis`): Objects to be concatenated.
+        Returns:
+            :obj:`FDataGrid`: FDataGrid object with the samples from the
+            original objects.
+        Raises:
+            ValueError: In case the provided list of FDataBasis objects is
+            empty.
+        Todo:
+            By the moment, only unidimensional objects are supported in basis
+            representation.
+        """
+        objects = iter(objects)
+        first = next(objects, None)
+
+        if not first:
+            raise ValueError("At least one FData object must be provided "
+                             "to concatenate.")
+
+        return first.concatenate(*list(objects))
```

Comment:
Could it be possible not to convert it to a list?

Reply:
Yes, my bad. Fixed